### PR TITLE
A couple of minor tweaks

### PR DESCRIPTION
**Pull request by NiLuJe**
ScreenSaver: Possibly avoid crashes in folder mode on devices that might have been used on macOS.
UIManager: Simplify the mechanism added in #9173

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/9200)

### CODE REVIEW

**Comment by NiLuJe** [comment]
(No squashing!)

File: frontend/ui/screensaver.lua [hunk 1]
```diff
@@ -23,6 +23,7 @@ local logger = require("logger")
 local _ = require("gettext")
 local ffiUtil = require("ffi/util")
 local T = ffiUtil.template
+local util = require("util")
```
Comment (Frenzie) [inline frontend/ui/screensaver.lua:26]: Pssht should go up a bit (ffiUtil, lfs, logger, util, _, T) if I'm not mistaken.

File: frontend/ui/screensaver.lua [hunk 2]
```diff
@@ -13,17 +13,18 @@ local InfoMessage = require("ui/widget/infomessage")
 local ImageWidget = require("ui/widget/imagewidget")
 local Math = require("optmath")
 local OverlapGroup = require("ui/widget/overlapgroup")
+local Screen = Device.screen
```
Comment (Frenzie) [inline frontend/ui/screensaver.lua:16]: Ah no, read our code style again, heh (or we could change it I suppose).

**Review by Frenzie — APPROVED** [review]
:p

**Pull request merged**